server/opts.go

```diff
@@ -81,20 +81,20 @@ type ClusterOpts struct {
 // NOTE: This structure is no longer used for monitoring endpoints
 // and json tags are deprecated and may be removed in the future.
 type GatewayOpts struct {
-	Name             string               `json:"name"`
-	Host             string               `json:"addr,omitempty"`
-	Port             int                  `json:"port,omitempty"`
-	Username         string               `json:"-"`
-	Password         string               `json:"-"`
-	AuthTimeout      float64              `json:"auth_timeout,omitempty"`
-	TLSConfig        *tls.Config          `json:"-"`
-	TLSTimeout       float64              `json:"tls_timeout,omitempty"`
-	TLSMap           bool                 `json:"-"`
-	TLSImplicitAllow bool                 `json:"-"`
-	Advertise        string               `json:"advertise,omitempty"`
-	ConnectRetries   int                  `json:"connect_retries,omitempty"`
-	Gateways         []*RemoteGatewayOpts `json:"gateways,omitempty"`
-	RejectUnknown    bool                 `json:"reject_unknown,omitempty"` // config got renamed to reject_unknown_cluster
+	Name               string               `json:"name"`
+	Host               string               `json:"addr,omitempty"`
+	Port               int                  `json:"port,omitempty"`
+	Username           string               `json:"-"`
+	Password           string               `json:"-"`
+	AuthTimeout        float64              `json:"auth_timeout,omitempty"`
+	TLSConfig          *tls.Config          `json:"-"`
+	TLSTimeout         float64              `json:"tls_timeout,omitempty"`
+	TLSMap             bool                 `json:"-"`
+	TLSAcceptKnownUrls bool                 `json:"-"`
```

Comment:
For some reason, I feel like this name sounds less restrictive when enabled than when it is disabled, which is not the case. Meaning that the default is `false`, which sounds like the server is not accepting something, while in reality, when false it means that it may accept things that it should not.

Would `TLSCheckKnownUrls` be better? I know we agreed on `verify_and_accept_known_urls` but maybe we could also change that one with `verify_and_check_known_urls`. If the `verify_and_check_` seem strange, maybe having `verify_cert_and_check_known_urls` lays out explicitly what this is about.